datasets/semeval18_emotion_classification/semeval18_emotion_classification.py

```diff
@@ -0,0 +1,154 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import datasets
+
+
+_CITATION = """\
+@InProceedings{SemEval2018Task1,
+ author = {Mohammad, Saif M. and Bravo-Marquez, Felipe and Salameh, Mohammad and Kiritchenko, Svetlana},
+ title = {SemEval-2018 {T}ask 1: {A}ffect in Tweets},
+ booktitle = {Proceedings of International Workshop on Semantic Evaluation (SemEval-2018)},
+ address = {New Orleans, LA, USA},
+ year = {2018}}
+"""
+
+_DESCRIPTION = """\
+ SemEval-2018 Task 1: Affect in Tweets: SubTask 5: Emotion Classification.
+ This is a dataset for multilabel emotion classification for tweets.
+ 'Given a tweet, classify it as 'neutral or no emotion' or as one, or more, of eleven given emotions that best represent the mental state of the tweeter.'
+ It contains 22467 tweets in three languages manually annotated by crowdworkers using Best–Worst Scaling.
+"""
+
+_HOMEPAGE = "https://competitions.codalab.org/competitions/17751"
+
+_LICENSE = ""
+
+_URLs = {
+    "english": [
+        "http://saifmohammad.com/WebDocs/AIT-2018/AIT2018-DATA/E-c/English/2018-E-c-En-train.zip",
+        "http://saifmohammad.com/WebDocs/AIT-2018/AIT2018-DATA/E-c/English/2018-E-c-En-dev.zip",
+        "http://saifmohammad.com/WebDocs/AIT-2018/AIT2018-DATA/AIT2018-TEST-DATA/semeval2018englishtestfiles/2018-E-c-En-test.zip",
+    ],
+    "spanish": [
+        "http://saifmohammad.com/WebDocs/AIT-2018/AIT2018-DATA/E-c/Spanish/2018-E-c-Es-train.zip",
+        "http://saifmohammad.com/WebDocs/AIT-2018/AIT2018-DATA/E-c/Spanish/2018-E-c-Es-dev.zip",
+        "http://saifmohammad.com/WebDocs/AIT-2018/AIT2018-DATA/AIT2018-TEST-DATA/semeval2018spanishtestfiles/2018-E-c-Es-test.zip",
+    ],
+    "arabic": [
+        "http://saifmohammad.com/WebDocs/AIT-2018/AIT2018-DATA/E-c/Arabic/2018-E-c-Ar-train.zip",
+        "http://saifmohammad.com/WebDocs/AIT-2018/AIT2018-DATA/E-c/Arabic/2018-E-c-Ar-dev.zip",
+        "http://saifmohammad.com/WebDocs/AIT-2018/AIT2018-DATA/AIT2018-TEST-DATA/semeval2018arabictestfiles/2018-E-c-Ar-test.zip",
+    ],
+}
+
+
+class SemEval18EmotionClassification(datasets.GeneratorBasedBuilder):
```

Comment:
If you rename the dataset `sem_eval_2018_task_5` you will have to rename this class `SemEval2018Task5`